presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -116,6 +117,8 @@
     private final WarningCollector warningCollector;
     private final FunctionResolution functionResolution;
 
+    private static final double LOWEST_APPROX_DISTINCT_MAX_STANDARD_ERROR = 0.023;
```

Comment:
remove this one; the default value can be directly obtained in class `DefaultApproximateCountDistinctAggregation`. Make `DEFAULT_STANDARD_ERROR` public in that class